packages/web-react/src/components/Modal/demo/index.tsx

```diff
@@ -27,6 +28,9 @@ ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
       <DocsSection title="Disabled Backdrop Click">
         <ModalDisabledBackdropClick />
       </DocsSection>
+      <DocsSection title="Hidden Close Button">
```

Comment:
It is not just a hidden close button, it is also a disabled escape key. Maybe something like `Dialog choice` or similar naming for simple actions like yes/no, confirm/cancel.